store.html

```diff
@@ -0,0 +1,191 @@
+
+<!DOCTYPE >
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="store.css">
+
+
+
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+
+</head>
+
+<body>
+    <!--header-->
+    <header>
+        <img src="img/karma-logo.svg" alt="karma logo" class="logo" />
+    
+        <!--navbar-->
+        <nav>
+            <ul>
+                <li>
+                    <a href="meetkarma">
```

Comment:
You can a link to the home  page here, while the other href tags should be set at "#"
```suggestion
                    <a href="index.html">
```